### PR TITLE
Symbol type

### PR DESCRIPTION
**Pull request by mahdiHashemi14**
Please review and comment / commit if anything is wrong

### CODE REVIEW

File: 1-js/04-object-basics/08-symbol/article.md [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
+با توجه به خصوصیات زبان، کلیدهای ویژگی‌های شیء می‌توانند یا از نوع رشته باشند، یا از نوع سمبل (symbol). نه از نوع اعداد و نه boolean، فقط رشته یا سمبل، همین دو نوع.
 
-Till now we've been using only strings. Now let's see the benefits that symbols can give us.
+تا اینجا ما فقط از رشته‌ها استفاده می‌کردیم. حال بیایید مزایایی که سمبل‌ها به ما می‌دهند را ببینیم.
 
-## Symbols
+## سمبل‌ها (Symbols)
 
-A "symbol" represents a unique identifier.
+یک "سمبل" نشان دهنده‌ی شناسه‌ای یکتا است.
```
Comment (mahdyar) [inline 1-js/04-object-basics/08-symbol/article.md:10]: For quoting in Persian, we use "«»", not double-quotations.
```suggestion
یک «سمبل» نشان دهنده‌ی شناسه‌ای یکتا است.
```

File: 1-js/04-object-basics/08-symbol/article.md [hunk 2]
```diff
@@ -1,31 +1,31 @@
 
-# Symbol type
+# نوع سمبل
```
Comment (mahdyar) [inline 1-js/04-object-basics/08-symbol/article.md:2]:
```suggestion
# نوع سمبل (Symbol type)
```

File: 1-js/04-object-basics/08-symbol/article.md [hunk 3]
```diff
@@ -70,80 +70,80 @@ alert(id.description); // id
 
 ````
 
-## "Hidden" properties
+## ویژگی‌های "مخفی"
```
Comment (mahdyar) [inline 1-js/04-object-basics/08-symbol/article.md:73]:
```suggestion
## ویژگی‌های «مخفی»
```

File: 1-js/04-object-basics/08-symbol/article.md [hunk 4]
```diff
@@ -176,60 +176,60 @@ let clone = Object.assign({}, user);
...
+هیچ تناقضی اینجا وجود ندارد. طراحی آن اینگونه است. زمانی که ما یک شیء را شبیه‌سازی می‌کنیم یا شیءها را ادغام می‌کنیم، معمولا می‌خواهیم که *تمام* ویژگی‌ها کپی شوند (شامل سمبل‌هایی مثل `id`).
 
-## Global symbols
+## سمبل‌های Global
 
-As we've seen, usually all symbols are different, even if they have the same name. But sometimes we want same-named symbols to be same entities. For instance, different parts of our application want to access symbol `"id"` meaning exactly the same property.
+همانطور که دیدیم، معمولا تمام سمبل‌ها متفاوت هستند، حتی اگر اسم یکسان داشته باشند. اما گاهی اوقات ما می‌خواهیم که سمبل‌های هم نام با یکدیگر برابر باشند. برای مثال، قسمت‌های مختلف برنامه ما می‌خواهند به سمبل `"id"` که ویژگی یکسانی است دسترسی داشته باشند.
 
-To achieve that, there exists a *global symbol registry*. We can create symbols in it and access them later, and it guarantees that repeated accesses by the same name return exactly the same symbol.
+برای بدست آوردن آن، یک *دفتر ثبت سمبل global* وجود دارد. ما می‌توانیم سمبل‌ها را درون آن بسازیم و بعدا به آن دسترسی داشته باشیم، و تضمین می‌کند که دسترسی تکراری با استفاده از یک اسم به ما دقیقا سمبل یکسان را برمی‌گرداند.
```
Comment (mahdyar) [inline 1-js/04-object-basics/08-symbol/article.md:185]:
```suggestion
برای بدست آوردن آن، یک *ریجستجری ثبت سمبل global* وجود دارد. ما می‌توانیم سمبل‌ها را درون آن بسازیم و بعدا به آن دسترسی داشته باشیم، و تضمین می‌کند که دسترسی تکراری با استفاده از یک اسم به ما دقیقا سمبل یکسان را برمی‌گرداند.
```

**Review by mahdyar — CHANGES_REQUESTED** [review]

**Comment by mahdiHashemi14** [comment]
Thank you for your review @mahdyar 
/done

**Review by mahdyar — APPROVED** [review]
Thank you, @mahdiHashemi14!

**Pull request merged**